infrahub_sdk/protocols.py

```diff
@@ -7,7 +7,8 @@
 from .protocols_base import CoreNode, CoreNodeSync
 
 if TYPE_CHECKING:
-    from .node import RelatedNode, RelatedNodeSync, RelationshipManager, RelationshipManagerSync
+    from infrahub_sdk.node import RelatedNode, RelatedNodeSync, RelationshipManager, RelationshipManagerSync
```

Comment:
Was there some problem with the relative import here?

Reply:
I haven't touched this part, not sure why the import was relative in the first place
this is what the current code is generating